src/test/regress/expected/tablespace_ddl.out

```diff
@@ -0,0 +1,15 @@
+-- allow tablespaces with empty location so that we can
+-- 1) test LOCATION, but
+-- 2) not have to create a directory on the filesytem to match LOCATION
+SET allow_in_place_tablespaces = true;
+-- create a tablespace using WITH clause
+CREATE TABLESPACE regress_tblspacewith LOCATION '' WITH (random_page_cost = 3.0); -- ok
+-- see that the tablespace ddl is correctly returned
+SELECT pg_get_tablespace_ddl('regress_tblspacewith');
+                                     pg_get_tablespace_ddl
+------------------------------------------------------------------------------------------------
+ CREATE TABLESPACE regress_tblspacewith LOCATION 'pg_tblspc/17342' WITH (random_page_cost = 3);
```

Comment:
My latest difficulty is this: I need to test the `LOCATION` clause of `CREATE TABLESPACE`. My current attempt borrows a trick from the existing tablespace test code, where setting GUC `allow_in_place_tablespace` allows me to set `LOCATION ''`. However, the returned location is not `''`; the returned location is something like `'pg_tblspc/17342'`... except that last 5 digit number is different with each test run!

One solution might be to create a test directory on the test postgres instance's filesystem so that I can use that directory as the LOCATION. But I'm not sure if the test suite even allows that.

Perhaps another creative solution is for the test to call `pg_get_tablespace_ddl()` and test the result with a regex?